messaging-experiments.yaml

```diff
@@ -1116,3 +1116,171 @@
     isEnrollmentPaused: false
     experimentDocumentUrl: https://experimenter.services.mozilla.com/experiments/message-router-content-experiment-high-velocity-message-testing-cfr-to-pip/
     userFacingDescription: "Message Router Content Experiment: High Velocity Message Testing: CFR to PiP"
+- id: MULTI-STAGE-ABOUTWELCOME-V3-80-RELEASE
+  enabled: true
+  filter_expression: env.version >= '80.' && env.version < '81.' && env.channel == 'beta'
+  # The filter `!('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)` is used to exclude
+  # users who have switched channels or re-installed the browser without having a new profile
+  targeting: localeLanguageCode == "en" && [userId, "multi-stage-aboutwelcome-v3-80"]|bucketSample(0, 2000, 10000) &&
+    ((isFirstStartup && !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) || 'multi-stage-aboutwelcome-v3-80' in activeExperiments)
+  arguments:
+    slug: multi-stage-aboutwelcome-v3-80
+    userFacingName: "Multi-Stage About:Welcome V3 80 Release"
+    experimentDocumentUrl: https://experimenter.services.mozilla.com/experiments/multi-stage-aboutwelcome-v3-80-release/
+    userFacingDescription: Study to validate Multi-stage onboarding experience.
+    isEnrollmentPaused: false
+    branches:
+      - slug: control
+        ratio: 1
+        groups:
+          - aboutwelcome
+        value: {}
+      - slug: treatment-import
+        ratio: 1
+        groups:
+          - aboutwelcome
+        value:
+          id: msw-import
```

Comment:
@ncloudioj as discussed shortening message id to use prefix msw in v3 80 experiment

Reply:
Looks good, thanks for the heads up!